loginserver.js

```diff
@@ -53,10 +53,22 @@ var LoginServer = module.exports = (function () {
 	LoginServer.request = function (action, data, callback) {
 		return getLoginServer(action).request(action, data, callback);
 	};
+	var TimeoutError = LoginServer.TimeoutError = function (message) {
```

Comment:
It would be useful for identifying errors. Better than checking ``error.message`` imo.

In particular, so far it seems that sometimes two errors are tied together: a request timeout, and a socket hang up, due to some inner workings of ``req.abort()``. I am considering checking for ``error instanceof TimeoutError`` so that the callbacks don't get called twice. But I'd like to investigate the issue some more...

Still, I can take ``TimeoutError`` out until it actually gets used, if you require it.